text/0000-landing-page.md

```diff
@@ -0,0 +1,58 @@
+# Meta
+[meta]: #meta
+- Name: Create a community landing page
+- Start Date: 1-6-2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+Prospective contributors to the project don't have a central place to go for information. This RFC proposes the creation of a community or "landing page" repo at `buildpacks/community` with high level information about the project including key personnel, governance, guidelines for contributions, meeting schedules, etc.
+
+# Outcomes
+[outcomes]: #outcomes
+
+ - New contributors feel welcomed.
+ - A new contributor can easily find information they want about the project, can join the slack and WG meetings, and can quickly contribute pull-requests, RFCs, and issues.
+
+# Motivation
+[motivation]: #motivation
+
+A major part of the success of the CNB project will necessitate growing contributors beyond the current core. As new people learn about the project and want to contribute, we need a friendly central landing place where all the information _about_ the project is housed.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This RFC proposes the creation of a project landing page that will be house at `buildpacks/community`.
+It will have a central `README.md` that included the following information and links to documents that provide more details if and when that's necessary:
+
+- Purpose of the project and high level overview including some canonical talk from YouTube, etc
+- Technical overview, including guides to the various components of the projects and links to their respective repositories
+- Project and community overview, including key personnel, governance structures (links to `GOVERNANCE.md`), link to slack, calendar links, and links to a contribution guide (this document is likely sufficiently detailed as to necessitate a separate document below)
+- `CONTRIBUTING.md` which details the RFC process and various other pieces of information about how to get started contributing to the project
```

Comment:
We should probably detail the RFC process as a README on the rfcs repos and just link out there. This way it doesn't need to be documented in more than one spot.